packages/adapter-knex/lib/adapter-knex.js

```diff
@@ -78,8 +78,6 @@ class KnexAdapter extends BaseKeystoneAdapter {
         console.log('Knex adapter: Dropping database');
       }
       await this.dropDatabase();
-    } else {
-      return [];
     }
     return this._createTables();
```

Comment:
Am I wrong in reading that this return statement will never be hit?

Reply:
With the `return` on 81 and 82 this was indeed the case. The `return` for `createTables` was never being hit and hence the issue where no tables where created for Postgres people.